src/fstests/fstests.jinja2

```diff
@@ -36,6 +36,16 @@ class Job(BaseJob):
         except Exception as e:
             print('Kernel build error:', e)
 
+    def _run_smoke_tests(self):
+        try:
+            result = subprocess.run("kvm-xfstests smoke", check=True, shell=True)
+            if result.returncode == 0:
+                print(f"Tests run with success.")
```

Comment:
I'd rather come up with something like "Test run success.", but that's just a suggestion.

Reply:
updated.